src/common.cpp

```diff
@@ -954,6 +958,8 @@ void common_flip(struct game_obj *game_object)
 	time_t last_seconds = last_frame.time;
 	struct game_mode *mode = getmode_cached();
 
+	if(aspect_ratio == AR_WIDESCREEN) widescreen.initParamsFromConfig();
```

Comment:
This seems an overkill. There is a better place to put this. There should be an initialize variables in ff7/field.cpp.

Reply:
That function is called at the start of field mode. But I'm not sure if the camera range are already available. Need to test it

Reply:
it worked fine, thanks